shelley/chain-and-ledger/dependencies/non-integer/doc/non-integer-calculations.tex

```diff
@@ -306,14 +404,15 @@ \subsection{Scaling}
   \right)
 \end{equation*}
 
-which guarantees that $\frac{x}{\exp(n)}$ is in the interval $[1; e]$ which lies
+This guarantees that $\frac{x}{\exp(n)}$ is in the interval $[1; e]$, which lies
 in the convergence radius of the approximation schemes.
 
 \subsection{Convergence}
 \label{sec:convergence}
 
-Experimental results have shown that continued fractions have a better
-convergence speed, in particular for the natural logarithm.
+Our experimental results have shown that continued fractions converge faster, in particular for the natural logarithm.
+
+\khcomment{Give captions for the plots.  Combine the plots from Figs. 1 and 2 into a single figure (one above the other?).  Do the same for Figs. 3 and 4.}
```

Comment:
The problem with this is that the vastly different y-axis will make one almost invisible, visually close to indistinguishable from the x-axis

Reply:
I meant one figure, with two plots...  Keep the actual plots as-is :)
subfigure sometimes works for this, but you may not need that.